requirements.in

```diff
@@ -1,4 +1,4 @@
-https://github.com/codecov/shared/archive/2db81dfa09d3884bb41307b639c9e7211cc62c41.tar.gz#egg=shared
+https://github.com/codecov/shared/archive/d8155cfdb5772eafb5127986dfe9041fca067e6e.tar.gz#egg=shared
```

Comment:
This is from a commit that's still in a PR, https://github.com/codecov/shared/pull/107, should I change this later to the one after it's merged?

Reply:
Updated, requirements.txt changed a bit, mostly ordering, after doing pip-complie not sure why but letting you know in case this is atypical